packages/web-components/src/tree-item/tree-item.template.ts

```diff
@@ -0,0 +1,72 @@
+import { children, elements, html, when } from '@microsoft/fast-element';
+import { FluentDesignSystem } from '../fluent-design-system.js';
+import type { TreeItem } from './tree-item.js';
+import { treeItemLevelToken } from './tree-item.style.js';
+
+// We don't put the icon into the icons/index.ts file because
+// this icon is the default chevron icon of the tree-item component,
+// it would be contributed to the upstream later.
+const chevronIcon = html`
+  <svg width="12" height="12" xmlns="http://www.w3.org/2000/svg" fill="currentColor">
+    <path
+      d="M4.65 2.15a.5.5 0 000 .7L7.79 6 4.65 9.15a.5.5 0 10.7.7l3.5-3.5a.5.5 0 000-.7l-3.5-3.5a.5.5 0 00-.7 0z"
+    ></path>
+  </svg>
+`;
+
+export const template = html<TreeItem>`
```

Comment:
Seems the template is missing many ARIA attributes, I'd recommend to check the ARIA Authoring Guide: https://www.w3.org/WAI/ARIA/apg/patterns/treeview/ to add the missing/necessary ARIA attributes.

Reply:
So the properties mentioned in this section https://www.w3.org/WAI/ARIA/apg/patterns/treeview/#wai-ariaroles,states,andproperties should be added?